tests/Feature/GetLinksTest.php

```diff
@@ -61,14 +61,14 @@ function get_single_link_by_link_id()
                 ->json('GET', '/api/links/all/'.$random_link_selection);
 
         $response->assertStatus(200);
-
-        $this->assertEquals(1, count($response->original));
+        $link = Link::where('id', $random_link_selection);
+        $this->assertEquals(1, count($link));
     }
```

Comment:
this test should verify that the returned response contained the expected link. Right now you are just verifying that the query builder (you need to add ->get()) is equal to one.. which i guess it would be... 

what i would do is verify that the returned json has the id of $random_link_selection cause you don't want to test that the Link::where()->get() works.. i would only do that if i am creating a link (POST data) and need to verify that it is in the db